tests/resources/mhd_window/image3x4-only-center-array.mhd

```diff
@@ -0,0 +1,17 @@
+ObjectType = Image
+NDims = 2
+BinaryData = True
+BinaryDataByteOrderMSB = False
+CompressedData = True
+CompressedDataSize = 11
+TransformMatrix = 1 0 0 1
+Offset = 0 0
+CenterOfRotation = 0 0
+ElementSpacing = 1 1
+DimSize = 3 4
+AnatomicalOrientation = ??
+ElementNumberOfChannels = 3
+WindowCenter = [10.5]
+WindowWidth = 100.5
```

Comment:
Curious: Is this what we actually want here? We support both forms? Arrays _and_ bare float values in one value with only implicit typing? I would plea to move this into its own value please, otherwise we get worse-than-DICOM madness in our MHD files, which is why we deliberately tried to avoid.

Also CIRRUS will just not load this right now, I am quite sure. Maybe add a new "WindowCenterList" field that is always a list? Or just remove the old one for the new value lists?

Reply:
I agree with this. I think it would save a lot of hassle if we fix the inconsistency in the dicom tags in the conversion step, rather than propagate it.

Reply:
Too late. Both forms exist already, and both are loaded by CIRRUS.

Reply:
> Maybe add a new "WindowCenterList" field that is always a list?

Absolutely not. We are not in the game of defining our own set of DICOM-like tags.

Reply:
Just to clarify: having `WindowWidth` and `WindowCenter` in different formats within one file is currently invalid.

Reply:
> I agree with this. I think it would save a lot of hassle if we fix the inconsistency in the dicom tags in the conversion step, rather than propagate it.

@MikeOverkamp-diag, are you suggesting an active approach to reformat `[10]/100` to `10/100`? I considered it but thought it safer to keep actual value manipulation to a minimum.

Reply:
Well... as is mentioned, the train has departed. So we sail with what we have right now. On the upside: We have a blueprint for repeated DICOM tag values now! Let's hope that nobody wrote array-like strings into a DICOM field :P